infra/modules/aws/main.tf

```diff
@@ -25,7 +25,7 @@ resource "aws_apigatewayv2_stage" "live" {
   auto_deploy   = true
   access_log_settings {
     destination_arn = aws_cloudwatch_log_group.gateway-log.arn
-    format          = "$context.identity.sourceIp $context.identity.caller $context.identity.user [$context.requestTime] \"$context.httpMethod $context.resourcePath $context.protocol\" $context.status $context.responseLength $context.requestId $context.extendedRequestId"
+    format          = "$context.identity.sourceIp $context.identity.caller $context.identity.user [$context.requestTime] \"$context.httpMethod $context.resourcePath $context.protocol\" $context.status $context.responseLength $context.requestId $context.extendedRequestId $context.integrationErrorMessage"
```

Comment:
adds error message, helped clarify what was going on with the gateway <-> lambda mapping